mac

```diff
@@ -150,6 +150,8 @@ brew update
 
 brew_install_or_upgrade 'zsh'
 brew_install_or_upgrade 'git'
+brew_install_or_upgrade 'mongodb'
+brew_launchctl_restart 'mongodb'
```

Comment:
I assume a lean, most-common subset. We aren't using mongo on any apps (except an old fork on erbitt we used for McK), and would be inclined to leave it off for manual as-needed.